tests/passing-configs/index.js

```diff
@@ -14,8 +15,6 @@ export function filePathToConfigObj(filepath) {
   return {
     config,
     filepath,
-    name: path.parse(filepath).name,
+    name: pathParse(filepath).name
```

Comment:
Awesome. My baby's waking up now, but if you need any help, I'll try to be available to help you out! :D